pages/index.js

```diff
@@ -2,6 +2,7 @@ import Article from './posts/article';
 import Footer from './components/footer';
 
 export default function Index() {
+  console.log("Index rendering")
```

Comment:
Avoid using console.log statements in production code. It can clutter the console and potentially expose sensitive information.